colorCodeManual.c

```diff
@@ -0,0 +1,29 @@
+#include "main_prv.h"
+#include "main_declare.h"
+
+extern const int MAX_COLORPAIR_NAME_CHARS;
+extern int numberOfMajorColors;
+extern int numberOfMinorColors;
+/*
+ **********************************************************************************************************************
+ * Service name         : verifyNumberToPair
+ * Syntax               : void printRefernceManual(void)
+ * param[in]            : pairNumber
+ * param[in]            : MajorColor
+ * param[in]            : expectedMinor
+ * return               : None
+ * Description          : Program to print color code refernce manual
+ **********************************************************************************************************************
+ */
```

Comment:
Consistency: Sometimes comments are present, sometimes not.
But the comments are not useful... are they? They don't seem to give additional information. What do you think?